pages/wallet.js

```diff
@@ -288,6 +295,7 @@ function InvoiceScanner ({ fieldName }) {
                 } else {
                   toaster.danger(error?.message || error?.toString?.())
                 }
+                onClose()
```

Comment:
I added this close on Error as I was having trouble continuing to scan /resetting the result on an error. Curious if you think the react-qr-scanner has an elegant way to reset result instead of doing this?

Reply:
I briefly tried a setInterval flow for retrying but this did not work as expected so I think for now we should stick with this